### ISSUE TITLE
Ignore permission "essentials.god" if not present

### ISSUE THREAD

**Issue by TNTUP** [EssentialsX/Essentials#1687]
EssentialsX version (`/essentials`): 
06:41:59 Essentials: Using locale en_US
06:41:59 Essentials: Using Paper 1.12+ ServerListPingEvent methods
06:41:59 CONSOLE: [INFO] Essentials reloaded 2.0.1-b547.

Server software (`/version`): 
06:42:21 CONSOLE: [INFO] This server is running Paper version git-Paper-1270 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)
06:42:21 CONSOLE: [INFO] You are running the latest version

Server (logs/latest.log):
Don't think this is required, just outputting the player does not have the permission and it removes the god mode.

EssentialsX config (if applicable): 

No setting to ignore if the player does not have the permission but got enabled by an admin. I do always enable god on my alt accounts but everytime I connect them they get removed so I need to renable it again and I know I can give the permission for it, but I have a lot of alts and even some friends are god'ed and I don't want to give the permission but allowing to have the god enabled by an admin.

EDIT: Did a comment on this commit https://github.com/delbertina/Essentials/commit/9de1cc5c6ee82cf01f9bff6b89755ea54ddef644

Thanks. (sorry I did search "god" and I guess im the only one who have that question?)

EDIT: For the time being, rollbacking to the last good build 510 I had backed up. If its not possible I'd be stuck on that build forever

EDIT2: Welp the server crashed due to invalid spawns/jails in build 510 on 1.12.2.. and I won't go to the last build due to this issue :/

EDIT3: Forked and removed that portion of code. Guessing that I can close this now, sorry if I've wasted any time!

**Comment by mdcfe** [comment]
Not quite sure why you picked that particular fork to comment on.

Regardless, this is the entire point of that feature.

Personally, removing this behaviour seems like a bad idea as it's very easy to forget whether a user has god enabled when they log off, but that's down to personal preference.

**Comment by TNTUP** [comment]
For my own server I don't care, because I set /god to players when I want and I don't want to give the permission for and removing it at anytime. Earlier builds of Essentials didn't had that. I'd suggest like an option "ignore-god-players-with-no-perm: true". Even I /god my accounts I own because I'm lazy to give the permission individually :) 

I don't know how to make patches like every upstream updates it will reapply the change I did for my own server^^
